src/pages/en/reference/api-reference.md

```diff
@@ -140,6 +140,11 @@ const data = await Astro.glob<CustomDataFile>('../data/**/*.js');
 ```
 
 See also: [`Astro.url`](#astrourl)
+
+:::note
+With the default `output: 'static'` configuration, `Astro.request.url` does not contain search parameters, like `?foo=bar`, as it's not possible to determine them ahead of time during static builds. However in `output: 'server'` mode, `Astro.request.url` does contain search parameters as it can be determined from a server request.
```

Comment:
Nitpick: In the Configuration Reference we call `output` an option, rather than a configuration, so I think we should change this for consistency. Though, I would love to know if there's any argument to choose configuration over option here.

```suggestion
With the default `output: 'static'` option, `Astro.request.url` does not contain search parameters, like `?foo=bar`, as it's not possible to determine them ahead of time during static builds. However in `output: 'server'` mode, `Astro.request.url` does contain search parameters as it can be determined from a server request.
```

Reply:
`option` sounds good to me!